addons/l10n_dk/data/account_tax_template_data.xml

```diff
@@ -567,37 +789,40 @@
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8740'),
+                'account_id': ref('a8785'),
+                'minus_report_line_ids': [ref('l10n_dk.account_tax_report_line_deduction_co2_tax')],
             }),
             (0,0, {
                 'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8730'),
+                'account_id': ref('a4271'),
             }),
         ]"/>
     </record>
-    <record id="tax620" model="account.tax.template">
+    <!-- water tax: 6.18 per cubic meter -->
+    <!-- https://skat.dk/data.aspx?oid=2246453 -->
+    <record id="tax760" model="account.tax.template">
         <field name="chart_template_id" ref="dk_chart_template"/>
-        <field name="sequence">620</field>
-        <field name="name">3. Land Ydelseskøb</field>
+        <field name="sequence">760</field>
+        <field name="name">Vandafgift</field>
         <field name="description"></field>
-        <field name="amount">25</field>
-        <field name="amount_type">percent</field>
+        <field name="amount">6.1800</field>
+        <field name="amount_type">fixed</field>
```

Comment:
same here. We are not going to add a line for every liter of water.

Reply:
As discussed it monday evening: the resulting tax will be `quantity * fixed_tax`.